### ISSUE TITLE
Key bindings

### ISSUE THREAD

**Issue by cpayne22** [BitfoxCanada/bruh-io-bug-tracker#52]
Any plans to be able to change the default keybinds?

I'm left handed, and the wasd / e / r combination doesn't work.

Thanks!

**Comment by RetroGamer24** [comment]
Yes, the ability to change them in the settings menu would be great, and I really wish this would be available. I think it would also be pretty cool to be able to use your `middle mouse button` to switch weapons by scrolling it, and an option to also use it for stuff like meds, or the `tab button` would switch through them.
